prisma/schema.prisma

```diff
@@ -271,13 +271,13 @@ model MaspTransaction {
   main             Boolean
   event            Event?
 
-  @@unique([transaction_hash], name: "uq_masp_on_transaction_hash", map: "uq_masp_on_transaction_hash")
-  @@map("masp_transactions")
+  @@unique([transaction_hash, asset_name, type], name: "uq_multi_asset_on_transaction_hash_asset_name_type", map: "uq_multi_asset_on_transaction_hash_asset_name_type")
```

Comment:
I may have missed the conversation on this, but I think we can technically have [multiple mints](https://github.com/iron-fish/ironfish/blob/staging/ironfish-rust/src/transaction/mod.rs#L152-L156) per transaction hash for the same asset. Can we just use an index on `transaction_hash`?

Reply:
Nevermind, I talked to Jason. We can leverage this to help us with the weekly checks and it'll make the upsert logic easier